src/webui/www/private/scripts/mocha-init.js

```diff
@@ -915,6 +916,31 @@ const initializeWindows = function() {
         return names.join("\n");
     };
 
+    copyInfohashFN = function(policy) {
```

Comment:
You could define an enum-like object and use it if you want: https://stackoverflow.com/a/5040502

Reply:
> 
> 
> You could define an enum-like object and use it if you want: https://stackoverflow.com/a/5040502

I don't want.
I wanted to do this initially, but then I realized that enum in JS is "yet another workaround". People don't even have a consensus on how it should be done correctly. I don't want to get involved with this.